docs/source/etl/variant-normalization.rst

```diff
@@ -57,17 +63,27 @@ The ``normalize_variants`` transformer has the following options:
    * - Option
      - Type
      - Possible values and description
-   * - ``referenceGenomePath``
+   * - ``reference_genome_path``
      - string
-     - Path to the reference genome ``.fasta`` or ``.fa`` file (required for normalization)
-
-       **Note**: ``.fai`` and ``.dict`` files with the same name must be present in the same folder.
-
+     - Path to the reference genome ``.fasta`` or ``.fa`` file. This file must be accompanied with a .fai index file in the same folder.
+   * - ``replace_columns``
+     - boolean
+     - |
+       | ``False``: The transformer does not touch the original ``start``, ``end``, ``referenceAllele`` and ``alternateAlleles`` columns. Instead, a StructType column called ``normalizationResult`` is added to the DataFrame which contains the normalized ``start``, ``end``, ``referenceAllele``, and ``alternateAlleles`` columns as well as the ``normalizationStatus`` StructType as the fifth field, which contains the following subfields:
+       |        ``changed``: Indicates whether the variant data was changed as a result of normalization or not.
+       |        ``errorMessage``: An error message in case the attempt at normalizing the row hit an error. In this case, the ``changed`` field will be set to ``false``. If no errors occur this field will be ``null``. In case of error, the first four fields in ``normalizationResult`` will be ``null``.
```

Comment:
We should be consistent with whether boolean values are capitalized (Scala vs Python style).

Reply:
Done